lib/humanfilesize.ts

```diff
@@ -2,7 +2,11 @@ import { getCanonicalLocale } from '@nextcloud/l10n'
 
 const humanList = ['B', 'KB', 'MB', 'GB', 'TB'];
 
-export function formatFileSize(size: number, skipSmallSizes: boolean = false): string {
+export function formatFileSize(size: number|string, skipSmallSizes: boolean = false): string {
+
+	if (typeof size === 'string') {
```

Comment:
You were testing with strings in tests, I figured it was best to fix the error and adapt the method @ChristophWurst 
Feel free to ask me if you prefer me to rollback

Reply:
I think this was a copy of the server code